pcap.c

```diff
@@ -592,6 +592,7 @@ pcap_create_common(char *ebuf, size_t size)
 	p->opt.buffer_size = 0;			/* use the platform's default */
 	p->opt.promisc = 0;
 	p->opt.rfmon = 0;
+	p->opt.rfmon_selfstart = 0;
```

Comment:
It has been fixed at: https://github.com/the-tcpdump-group/libpcap/pull/512/commits/1c3bf75b09e74efaf63179ef8aba000d2610b415